src/client/pages/CountryHome/PanEuropeanHome/hooks/useSections.tsx

```diff
@@ -6,7 +6,7 @@ import { Users } from 'meta/user'
 import { useCycle } from 'client/store/assessment'
 import { useUser } from 'client/store/user'
 import { useCountryIso } from 'client/hooks'
-import Collaborators from 'client/pages/CountryHome/FraHome/Collaborators'
+import Collaborators from 'client/pages/CountryHome/Collaborators'
```

Comment:
Collaborators page moved one level higher as it is shared by both PanEuropean and FRA Assessments